app/data-sources/authenticate/AuthenticateDatabase.js

```diff
@@ -35,40 +36,52 @@ export class AuthenticateDatabase {
     Updated: DataTypes.DATE
   })
 
+  constructor (config = {}) {
+    this.logger = config.logger
+  }
+
   async healthCheck () {
     try {
+      this.logger.verbose('#datasource - authenticate - checking database connection', { code: AUTHENTICATE_DATABASE_READ_001 })
       await this.dbRead.authenticate()
-      logger.health('#authenticate - database authenticated', { code: AUTHENTICATE_DATABASE_ALL_001 })
+      this.logger.debug('#datasource - authenticate - database connection established', { code: AUTHENTICATE_DATABASE_READ_001 })
     } catch (error) {
-      logger.error('Authenticate database error', { error, code: AUTHENTICATE_DATABASE_ALL_001 })
+      this.logger.error('#datasource - authenticate - connection error', { error, code: AUTHENTICATE_DATABASE_READ_001 })
       throw error
     }
   }
 
   async getAuthenticateQuestionsAnswersByCRN (crn, employeeId) {
     try {
-      logger.silly('Creating audit record in authenticate database', { employeeId, crn })
+      this.logger.verbose('#datasource - authenticate - Creating audit record in authenticate database', { employeeId, crn, code: AUTHENTICATE_DATABASE_WRITE_001 })
```

Comment:
any reason this is verbose rather than debug as above?

Reply:
I have put verbose as the entering log, which will show just the id, and debug is the exit log that shows the response in detail. probably wont have debug on higher envs to prevent certain data in responses being logged? 

Was my thinking anyway

Reply:
just looked and looks like these are the default levels are:

```
error
warn
info
http
verbose
debug
silly
  ```